cli/command/container/prune_test.go

```diff
@@ -20,5 +22,8 @@ func TestContainerPrunePromptTermination(t *testing.T) {
 		},
 	})
 	cmd := NewPruneCommand(cli)
-	test.TerminatePrompt(ctx, t, cmd, cli, nil)
+	test.TerminatePrompt(ctx, t, cmd, cli, func(t *testing.T, err error) {
+		t.Helper()
+		assert.ErrorIs(t, err, command.ErrPromptTerminated)
+	})
```

Comment:
Is this ... reverting the changes from the previous commit? Should those be squashed?

Reply:
Yes sorry! I am very bad at keeping code organized across commits. I would prematurely commit something, realize it's bad and then revert with another commit. It's new to me to squash commits in my branch for review purposes, I'll improve this for future PRs